base/TorqueRobotBase.java

```diff
@@ -35,7 +31,7 @@
  *
  * @author Justus Languell
  */
-public class TorqueRobotBase extends LoggedRobot {
+public class TorqueRobotBase extends TimedRobot {
```

Comment:
I cannot merge because this is necessary

Reply:
yes, sorry this was an accident. I didn't have the advantage kit stuff in the build.gradle and I just removed the AK references. I'll undo this.